api/src/test/java/au/org/aodn/nrmn/restapi/service/SpreadSheetServiceIT.java

```diff
@@ -125,6 +124,18 @@ public void shortInvertsRowShouldBeInvalid() throws Exception {
         }
         assertTrue(StringUtils.isNotEmpty(error));
     }
+    /**
+     * Excel row2 must be blank according to the import file spec, if not blank throw exception
+     */
+    @Test
+    void rejectFileWhenRow2IsNotEmpty() {
+        Exception e = assertThrows(Exception.class, () -> {
+            FileSystemResource file3 = new FileSystemResource("src/test/resources/sheets/row2NotEmpty.xlsx");
+            ParsedSheet parsedSheet = sheetService
```

Comment:
Unused variable?

Reply:
Yes, removed as the code expected exception